scripts/werf-in-image/werf-argocd-cmp-sidecar.yaml

```diff
@@ -4,4 +4,16 @@
 {{ $_ := set . "DefaultChannelName" "stable" }}
 {{ $_ := set . "DefaultDistroName" "ubuntu" }}
 
+# set up default channel version  
+{{- $trdlChannelsValues := .Files.Get "../../trdl_channels.yaml" | fromYaml -}}
+{{- range $_, $groupObject := $trdlChannelsValues.groups }}
+  {{- if eq $groupObject.name $.DefaultGroupName }}
+    {{- range $_, $channelObject := $groupObject.channels }}
+      {{- if eq $channelObject.name $.DefaultChannelName}}
+        {{ $_ := set $ "DefaultChannelVersion" $channelObject.version }}
+      {{- end}}
+    {{- end}}
+  {{- end}}
+{{- end}}
```

Comment:
Why this code is not common (not in `scripts/werf-in-image/.werf/templates/base_config.tmpl`)?